Images/grassTop.png.import

```diff
@@ -0,0 +1,32 @@
+[remap]
```

Comment:
Issue with textures, possibly with the image file or maybe with import settings? Have a look at the other import settings, maybe turn off sampling/mipmaps or some shit ?
![image](https://user-images.githubusercontent.com/11905377/36425681-6b7e4c58-163f-11e8-9c9a-29b00ab88638.png)

Reply:
I think we have this bug on all textures, just it's more noticeable against the green. Probably not a big issues, try offseting UVs by a small value epsilon?

Reply:
I'm not sure how to do this.

Reply:
Ok, it's my problem them, will investigate. We can ignore this issue for this PR.